sphinx_gallery/gen_gallery.py

```diff
@@ -135,28 +152,55 @@ def touch_empty_backreferences(app, what, name, obj, options, lines):
         open(examples_path, 'w').close()
 
 
-gallery_conf = {
-    'filename_pattern': re.escape(os.sep) + 'plot',
-    'examples_dirs': '../examples',
-    'gallery_dirs': 'auto_examples',
-    'mod_example_dir': os.path.join('modules', 'generated'),
-    'doc_module': (),
-    'reference_url': {},
-}
+def exit_on_fail_examples(app, exception):
+    """Embed hyperlinks to documentation into example code"""
+    if exception is not None:
+        return
+
+    gallery_conf = app.config.sphinx_gallery_conf
+    failing_examples = set(gallery_conf['failing_examples'])
+    ex_fail_examples = set(gallery_conf['expected_failing_examples'])
+
+    still_ex_fails = failing_examples.intersection(ex_fail_examples)
+    expected_fail_msg = []
+    if still_ex_fails:
+        expected_fail_msg.append("Known failing examples, still failing:")
+        for fail_example in still_ex_fails:
+            expected_fail_msg.append(fail_example + ' failed leaving traceback:\n' +
+                                     gallery_conf['failing_examples'][fail_example] + '\n')
+        warnings.warn("\n".join(expected_fail_msg))
```

Comment:
I would use a print statement rather than a warning ... The warning adds some not really necessary information:

```
/home/le243287/dev/sphinx-gallery/sphinx_gallery/gen_gallery.py:171: UserWarning: Known failing examples, still failing:
../examples/plot_raise.py failed leaving traceback:
Traceback (most recent call last):
  File "/home/le243287/dev/sphinx-gallery/sphinx_gallery/gen_rst.py", line 497, in execute_script
    exec(code_block, example_globals)
  File "<string>", line 2, in <module>
NameError: name 'iae' is not defined


  warnings.warn("\n".join(expected_fail_msg))
```